<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Figure Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="paintButton">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>408</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Paint</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>345</y>
      <width>79</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arbitrary</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="PerimetrOutput">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ArreaOutput">
    <property name="geometry">
     <rect>
      <x>706</x>
      <y>110</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CentreOutput">
    <property name="geometry">
     <rect>
      <x>706</x>
      <y>189</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>733</x>
      <y>83</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>4</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimetr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>164</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AngleInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZoomInput">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>479</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AnglePushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Angle rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZoomPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>479</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Set zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeCenter">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>278</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
